src/ElectricPokemon.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+//   2 private final variables
+    private final String name;
```

Comment:
We hebben je probleem met "null" waardes al besproken in een meeting. 
Hier hebben we geen "name" nodig, want die staat in de super. Wel kunnen we hier een "type" constante definieren.